plugins/azure_sentinel/plugin.spec.yaml

```
@@ -0,0 +1,879 @@
+        - Low
+        - Medium
+    status:
+      title: Incidents status
+      description: Incidents status
+      required: true
+      type: string
+      enum:
+        - Active
+        - Closed
+        - New
+    systemData:
+      description: Azure Resource Manager metadata containing createdBy and modifiedBy
+        information.
+      title: Azure Resource Manager metadata containing createdBy and modifiedBy information.
+      type: SystemData
+    title:
+      description: The title of the incident
+      title: The title of the incident
+      type: string
+    type:
+      description: Azure resource type
+      title: Azure resource type
+      type: string
+  Incident:
+      id:
+        title: Full Incident ID
+        description: Full incident ID
+        type: string
+      name:
+        title: Incident ID
+        description: Incident name - short id
+        type: string
+      type:
+        title: type
+        type: string
+      etag:
+        title: etag
+        type: string
+      properties:
+        title: Incident properties
+        description: Incident properties object.
+        type: IncidentProperties
+
+connection:
+  host:
+    title: Host
+    type: string
+    description: Azure REST API Server
+    required: true
+    default: https://management.azure.com
+  client_id:
+    title: Client ID
+    type: string
+    description: The application ID that the application registration portal assigned
+      to your app
+    required: true
+  client_secret:
+    title: Client Secret
+    type: credential_secret_key
+    description: The application secret that you generated for your app in the app
+      registration portal
+    required: true
+  tenant_id:
+    title: Tenant ID
+    type: string
+    description: This is active directory ID
+    required: true
+  api_version:
+    title: API Version
+    type: string
+    description: The version of the API to use. The current version is 2016-04-30-preview
+    required: true
+    default: 2016-04-30-preview
+actions:
+  create_update_incident:
+    title: Creates or updates an incident.
+    description: Creates or updates an incident. 
+    input:
+      incidentId:
+        title: Incident ID
+        required: true
+        description: Incident ID
+        type: string
+      resourceGroupName:
+        title: Resource Group Name
+        required: true
+        description: The name of the resource group within the user's subscription. The name is case insensitive.
+        type: string
+      subscriptionId:
+        title: Subscription ID
+        required: true
+        description: "Azure subscription ID Regex pattern: ^[0-9A-Fa-f]{8}-([0-9A-Fa-f]{4}-){3}[0-9A-Fa-f]{12}$"
+        type: string
+      workspaceName:
+        title: Workspace Name
+        required: true
+        description: The name of the workspace
+        type: string
+      etag:
+        title: etag
+        description: etag of the azure resource
```

```suggestion
        description: Etag of the Azure resource
```
Make sure to capitalize proper nouns - in this case, Azure should be capitalized.